### PR TITLE
blob/azureblob: add tests for OpenBucket and OpenURL

### PR DESCRIPTION
**Pull request by vangent**

### CODE REVIEW

File: blob/azureblob/azureblob_test.go [hunk 1]
```diff
@@ -123,3 +126,154 @@ func TestConformance(t *testing.T) {
...
+			if !test.nilServiceURL {
+				serviceURL, _ = ServiceURLFromAccountKey("x", "y")
+			}
+
+			// Create driver impl.
+			drv, err := openBucket(ctx, serviceURL, test.containerName, nil)
+			if (err != nil) != test.wantErr {
+				t.Errorf("got err %v want error %v", err, test.wantErr)
+			}
+			if drv != nil {
```
Comment (jba) [inline blob/azureblob/azureblob_test.go]: if err == nil && ...

File: blob/azureblob/azureblob_test.go [hunk 2]
```diff
@@ -123,3 +126,154 @@ func TestConformance(t *testing.T) {
...
+
+func TestOpenURL(t *testing.T) {
+
+	const (
+		accountName = "my-accoun-name"
+		accountKey  = "aGVsbG8=" // must be base64 encoded string, this is "hello"
+		sasToken    = "my-sas-token"
+	)
+
+	makeCredFile := func(name string, content string) *os.File {
```
Comment (jba) [inline blob/azureblob/azureblob_test.go]: name, content string

File: blob/azureblob/azureblob_test.go (hunk 1 above)
Comment (vangent) [reply blob/azureblob/azureblob_test.go]: Done.

File: blob/azureblob/azureblob_test.go (hunk 2 above)
Comment (vangent) [reply blob/azureblob/azureblob_test.go]: Done.

**Comment by vangent** [comment]
PTAL

**Pull request merged**